pkg/commands/git_commands/rebase.go

```diff
@@ -136,11 +141,17 @@ func (self *RebaseCommands) EditRebase(branchRef string) error {
 	}).Run()
 }
 
+type ChangeTodoAction struct {
+	sha       string
+	newAction todo.TodoCommand
+}
+
 type PrepareInteractiveRebaseCommandOpts struct {
-	baseShaOrRoot  string
-	todoLines      []TodoLine
-	overrideEditor bool
-	prepend        bool
+	baseShaOrRoot    string
+	todoLines        []TodoLine
+	overrideEditor   bool
+	prepend          bool
+	changeTodoAction []ChangeTodoAction
```

Comment:
Can we rename this to `changeTodoActions` or `todoActionChanges`?

Reply:
Yes, even though it goes away again at the end of the branch. Still, I like clean history, so I fixed it in the middle. Had to squash it in though, fixups wouldn't work for this.